### PR TITLE
Add a new XDR use case - Send notifications to a Webhook using playbooks

### PR DESCRIPTION
**Pull request by TonioRyo**

### CODE REVIEW

File: docs/xdr/usecases/playbook/notifications_using_playbooks.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+This use case describes how to send a notification to a third party system like Slack or Microsoft Teams using webhooks and a playbook.
+
+## Prerequisites
+
+- A Sekoia.io XDR licence
+- The user that configures the Playbook should have a Role that contains:
+	* The SYMPHONY permissions
+- An API Key with a Role that contains at least the following permission:
+	* SIC_READ_ALERTS
+    * SIC_READ_INTAKES
```
Comment (squioc) [inline docs/xdr/usecases/playbook/notifications_using_playbooks.md:12]:
```suggestion
	* SIC_READ_ALERTS
	* SIC_READ_INTAKES
```

File: docs/getting_started/notifications-Examples.md [hunk 2]
```diff
@@ -27,6 +27,7 @@ The “WebHook notification” will let you send message to interact with third
 
 !!! info 
     You can’t use the WebHook notification mechanism to push information directly to third parties (such as Slack or Telegram), you have to use an intermediate server. To do so, you can use solutions like IFTTT or a simple HTTP server (see below).
+    There are also playbook templates that can be used to send notifications to Slack or Microsoft Teams directly using Webhooks, see this documentation for more information [Send notifications to a Webhook using a playbook](../xdr/usecases/playbook/notifications_using_playbooks.md)
```
Comment (squioc) [inline docs/getting_started/notifications-Examples.md:30]:
```suggestion
    There are also playbook templates that can be used to send notifications to Slack or Microsoft Teams directly using Webhooks; See how to [send notifications to a Webhook using a playbook](../xdr/usecases/playbook/notifications_using_playbooks.md) for more information 
```

**Review by squioc — COMMENTED** [review]
I propose two improvements

**Review by squioc — APPROVED** [review]

File: docs/getting_started/notifications-Examples.md [hunk 3]
```diff
@@ -27,6 +27,7 @@ The “WebHook notification” will let you send message to interact with third
 
 !!! info 
     You can’t use the WebHook notification mechanism to push information directly to third parties (such as Slack or Telegram), you have to use an intermediate server. To do so, you can use solutions like IFTTT or a simple HTTP server (see below).
+    There are also playbook templates that can be used to send notifications to Slack or Microsoft Teams directly using Webhooks; See how to [send notifications to a Webhook using a playbook](../xdr/usecases/playbook/notifications_using_playbooks.md) for more information
```
Comment (ka0ula) [inline docs/getting_started/notifications-Examples.md:30]:
```suggestion
    There are also playbook templates that can be used to send notifications to Slack or Microsoft Teams directly using Webhooks. See how to [send notifications to a Webhook using a playbook](../xdr/usecases/playbook/notifications_using_playbooks.md) for more information.
```

File: docs/xdr/usecases/playbook/notifications_using_playbooks.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+	* The SYMPHONY permissions
+- An API Key with a Role that contains at least the following permission:
+	* SIC_READ_ALERTS
+	* SIC_READ_INTAKES
+
+> To create your API Key, follow this [documentation](../../../getting_started/manage_api_keys.md).
+
+## Create your playbook
+
+Playbook templates were created to ease the process of setting up the playbooks to send the notifications to Slack or Teams.
```
Comment (ka0ula) [inline docs/xdr/usecases/playbook/notifications_using_playbooks.md:18]:
```suggestion
Playbook templates were created to ease the process of setting up the playbooks to send the notifications to Slack or Teams. 

To create one, follow these steps: 

```

File: docs/xdr/usecases/playbook/notifications_using_playbooks.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+## Configure your playbook
+
+Once your playbook is created, the following configuration steps are required:
+
+- Open each `Sekoia.io` action and select the account that needs to be used to communicate with Sekoia.io APIs. You may have to create an account with the API key you created previously if no account exists.
+- Open the `Send to Slack` or `Send to Teams` action and replace the URL with the URL of your webhook
+
+## Customize your notifications
+
+The Slack and Teams playbook templates are provided as examples of notifications that can be sent to these solutions, you can customize the playbooks to your needs by adding additionnal actions to retrieve more data from the alerts and events and add them to the payload sent. Keep in mind that the payload must match the format expected by the third parties.
```
Comment (ka0ula) [inline docs/xdr/usecases/playbook/notifications_using_playbooks.md:36]:
```suggestion
The Slack and Teams playbook templates are provided as examples of notifications that can be sent to these solutions. You can customize the playbooks to suit your needs by adding additional actions to retrieve more data from the alerts and events and add them to the payload sent. 

Keep in mind that the payload must match the format expected by the third parties.
```

**Pull request merged**